WordPress/Classes/ViewRelated/What's New/Views/WhatIsNewViewController.swift

```diff
@@ -6,17 +6,26 @@ class WhatIsNewViewController: UIViewController {
 
     private let makeWhatIsNewView: () -> WhatIsNewView
 
+    private let onContinue: () -> Void
+    private let onDismiss: (() -> Void)?
```

Comment:
Sorry I'm late: should we make these two the same type, or is there a reason why one is optional and the other isn't?